File: webapp/HIN_WEB/ClientApp/src/app/model/organization.ts
import { LabelType } from "./label-type";

export class Organization {
  public Id: number;
  public LeadId: number;
  public Name: string;
  public LabelId: number;
  public Address: string;
  public UpdatedOn: Date;
  public UpdatedBy: string;
  public UpdatedById: number;
  public CreatedOn: Date;
  public CreatedBy: string;
  public CreatedById: number;
  public City: string;
  public Country: string;
  public State: string;
  public ZipCode: string;
  public Website: string;
  public Label: LabelType;
}

File: webapp/HIN_WEB/Models/Organization.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace HIN_WEB.Models
{
    public partial class Organization
    {
        public int Id { get; set; }
        public int? LeadId { get; set; }
        public string Name { get; set; }
        public int? LabelId { get; set; }
        public string Address { get; set; }
        public DateTime? UpdatedOn { get; set; }
        public string UpdatedBy { get; set; }
        public int? UpdatedById { get; set; }
        public DateTime? CreatedOn { get; set; }
        public string CreatedBy { get; set; }
        public int? CreatedById { get; set; }
        public string Website { get; set; }
        public string City { get; set; }
        public string State { get; set; }
        public string Country { get; set; }
        public string ZipCode { get; set; }
        public virtual LabelType Label { get; set; }
    }
}
